packages/core/package.json

```diff
@@ -103,6 +103,7 @@
         "@deriv/deriv-charts": "^2.0.0-beta.18",
         "@deriv/hooks": "^1.0.0",
         "@deriv/p2p": "^0.7.3",
+        "@deriv/account-v2": "^2.0.0",
```

Comment:
We already have a version 2? :)

Reply:
accounts-v2 is a temporary name. Once whole account package is coded inside account-v2. We will remove the v2 part. The version will be 2